src/controller/CommissionerDiscoveryController.cpp

```diff
@@ -34,6 +34,13 @@
 using namespace ::chip;
 using namespace chip::Protocols::UserDirectedCommissioning;
 
+CommissionerDiscoveryController gCommissionerDiscoveryController;
```

Comment:
So this forces on the core SDK level (unlike before, where it was in the example app), that there is a unique commissioner discovery controller.  Is that reasonable?  I don't have a good feel for how this gets used in practice by SDK consumers.

Reply:
reverted